app/mainwindow.cpp

```diff
@@ -923,12 +925,14 @@ void MainWindow::setupActions()
     SetActionShortcut(ui->actionNew, QKeySequence::New);
     SetActionShortcut(ui->actionOpen, QKeySequence::Open);
     SetActionShortcut(ui->actionSave, QKeySequence::Save);
-    ui->actionSave->setIcon(QIcon("fa-floppy-o.fontawesome"));
     SetActionShortcut(ui->actionSaveAs, QKeySequence::SaveAs);
-    ui->actionExportToPDF->setIcon(QIcon("fa-file-pdf-o.fontawesome"));
     SetActionShortcut(ui->action_Print, QKeySequence::Print);
-    ui->action_Print->setIcon(QIcon("fa-print.fontawesome"));
     SetActionShortcut(ui->actionExit, QKeySequence::Quit);
+#ifndef Q_OS_OSX
```

Comment:
Could you move all `setIcon()` calls to a separate private method? Then the Q_OS_OSX would be less spread. Thank you!